src/components/booksComponent/BooksList.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import BooksItem from './BooksItem';
+
+const BooksList = () => (
+  <div>
+    <h2>Books list</h2>
+    <BooksItem />
```

Comment:
- [ ] (Required) It would be nice to have a component that handles the list of books in this component and that component should have a hard-coded list of books for now with a non-functional `delete` button. For example, 

```
const SingleBook = () => (
<>
   <div>
      <h2>Name of Book</h2>
      <p>The Merchant of Venice</p>
      <button type="button">Delete</button>
    </div>
</>
)
```

Then in your `BookList Component`, you can do 
```
const BooksList = () => (
  <div>
    <SingleBook />
    <BooksItem />
  </div>
);
```

Although the hard-coded book item would be removed in a later milestone, it is necessary for you to include it to pass this milestone 👍 😄 .